lib/prmd/templates/schemata/link.md.erb

```diff
@@ -3,7 +3,7 @@
   response_example = link['response_example']
   link_schema_properties_template = Prmd::Template.load_template('link_schema_properties.md.erb', options[:template])
 -%>
-### <%= title %> <%= link['title'] %>
+### <a name="resource-<%= resource %>/<%= link['method'] %><%= link['href'] %>"><%= title %> <%= link['title'] %></a>
```

Comment:
For the name, I wonder if something like `link-#{method}-#{resource}` might be a bit clearer? So something like `link-get-app`? That said, it is common to have two GETs (one for individual and one for collection), which I don't think this would distinguish between?